### DISCUSSION TITLE
Meilisearch setup

### DISCUSSION THREAD

**Discussion by davide445** [FlowiseAI/Flowise#3405] [Q&A]
@MohamedYasserOaf I have tried to setup Meilisearch Cloud to work with Flowise.
Using local Ollama embeddings node (with BGE-m3 model), using this flow

[Meilisearch test Chatflow.json](https://github.com/user-attachments/files/17493717/Meilisearch.test.Chatflow.json)

Tried to create an index first and next upsert from Flowise and seems all is fine but as soon I try to chat i got the error message cannot find ollama embedding.
Tried to upsert from Flowose to a new index before creating it in Meilisearch and also in this case the process ended correctly on Flowise side, but I didn't see any index on Meilisearch.

Wanted to kindly ask if so far only local Meilisearch installation is supported from the Flowise node, or there is any other procedure I need to be aware of.

**Comment by MohamedYasserOaf** [comment]
@davide445 
Hi, glad to hear that you are using my module, would be more than happy if you join me on discord to further understand and debug your issue
